.gitignore

```diff
@@ -3,6 +3,7 @@
 .config
 .rvmrc
 .yardoc
+.ruby-version
```

Comment:
Somehow it was giving me more of a headache than helping. But I will revert that commit.